src/ontology/mondo-ingest.Makefile

```diff
@@ -331,10 +331,14 @@ reports/mirror_signature-%.tsv: component-download-%.owl $(COMPONENTSDIR)/%.db m
         python3 $(SCRIPTSDIR)/mirror_signature_via_oak.py --db-path $(COMPONENTSDIR)/$*.db --onto-config-path metadata/$*.yml --outpath $@;\
 	 else\
         $(ROBOT) query -i $(TMPDIR)/$<.owl --query ../sparql/classes.sparql $@;\
+        (head -n 1 $@ && tail -n +2 $@ | sort) > $@-temp;\
```

Comment:
FYI: Sorting of `reports/mirror_signature-%.tsv`

Reply:
Cool! Never new you could do that! Nice one.